build.gradle

```diff
@@ -26,6 +26,7 @@ allprojects {
     targetCompatibility = '1.8'
 
     repositories {
+        maven { url 'http://maven.aliyun.com/nexus/content/repositories/central/' }
```

Comment:
Is this needed? One would think Brave libs are on Maven central

Reply:
this is for speed up downloading package, I will delete it in next commit.